### DISCUSSION TITLE
Tooltip for longer event  description (notes)

### DISCUSSION THREAD

**Discussion by fabiobettiol** [wdelfuego/nova-calendar#67] [Ideas & Feature Requests]
Thank you for the great work in this tool. It's very useful.

Besides the name, times, notes and badge (all of them great, by the way) It would be nice to have an optional on-hover tooltip for displaying the notes. Due to the limited space, there may be and optional field in table with a longer description or further details about the event (without going to every resource to check).

**Comment by wdelfuego** [comment]
Thanks for creating this issue!

You can already add tooltips with long html content to [calendar day badges](https://wdelfuego.github.io/nova-calendar/customizing-the-calendar.html#adding-badges-to-calendar-day-cells), but I agree it'd be nice to have this functionality for calendar event badges or calendar events themselves. 

I'm open to pull requests; I think the functionality is quite simple to implement. If not I might pick this up for a future release.

Thanks again!
